weos_api.go

```diff
@@ -91,15 +91,27 @@ func (p *API) LogLevel(next echo.HandlerFunc) echo.HandlerFunc {
 		switch level {
 		case "debug":
 			p.EchoInstance().Logger.SetLevel(log.DEBUG)
+			cc.Logger().SetLevel(log.DEBUG)
 		case "info":
 			p.EchoInstance().Logger.SetLevel(log.INFO)
+			cc.Logger().SetLevel(log.INFO)
 		case "warn":
 			p.EchoInstance().Logger.SetLevel(log.WARN)
+			cc.Logger().SetLevel(log.WARN)
 		case "error":
 			p.EchoInstance().Logger.SetLevel(log.ERROR)
```

Comment:
This should probably be removed as per discussion since this is setting the log level application wide